boards/Makefile.common

```diff
@@ -164,7 +164,7 @@ endif
 # present, if not use the custom shipped one
 ifeq (, $(shell sha256sum --version 2>/dev/null))
   # No system sha256sum available
-  SHA256SUM := $(CARGO) run --manifest-path $(TOCK_ROOT_DIRECTORY)tools/sha256sum/Cargo.toml --
+  SHA256SUM := $(CARGO) run --manifest-path $(TOCK_ROOT_DIRECTORY)tools/sha256sum/Cargo.toml -- 2>/dev/null
```

Comment:
I only see this now, but throwing away all of the stderr output is overkill, and makes it harder to debug legitimate errors (i.e. failing to compile the `sha256sum` tool). Cargo supports a `--quiet` flag precisely for this use case (not printing the build steps, but still printing errors).